README.rst

```diff
@@ -57,6 +57,37 @@ can find it at `PULL_REQUEST_TEMPLATE.md <.github/PULL_REQUEST_TEMPLATE.md>`_.
 The issue report template should be automatically applied if you are creating an issue on GitHub as well. Otherwise you
 can find it at `ISSUE_TEMPLATE.md <.github/ISSUE_TEMPLATE.md>`_.
 
+Getting Started
+-----------------
+If you want to run dbt-schema-builder locally, clone this repository on your local system.
+You need a python virtual environment to run this, use the following commands to create and activate python 3.6.12 virtual environment
+::
+    $ pyenv install 3.6.12
+    $ pyenv virtualenv 3.6.12 schemabuilder-venv
+    $ pyenv activate schemabuilder-venv
+    $ cd dbt-schema-builder
+
+Next, you will run following commands
+::
+    $ pip install -U pip-tools
+    $ pip-compile --rebuild --upgrade  -o requirements/pip-tools.txt requirements/pip-tools.in
+
+Run the following to update the versions in all of the local requirements files    
+::
+    $ make upgrade
+
+The following command will install Schema Builder and dependencies installed in your virtualenv
+::
+    $ pip install .
+
+Now you will be all set to run dbt-schema-builder tool locally, cd into warehouse-transforms project directory in which you want to run dbt-schema-builder tool
```

Comment:
This is a public repo, so not everyone will have access to warehouse transforms. I think you can generalize this to "cd into the dbt project directory in which you want to run..."